docs/faq.rst

```diff
@@ -203,6 +204,18 @@ itself. GitHub provides a
 `tutorial <https://help.github.com/en/articles/approving-oauth-apps-for-your-organization/>`__
 for approving third party apps access to your organization.
 
+As you navigate GitHub's administrative interface, you may see a
+message indicating that you have denied Dockstore access
+to your organization:
+
+.. image:: /assets/images/docs/dockstore-oauth-access-denied.png
+
+To grant access to the Dockstore application, click the *Grant access* button.
+
+If you don't grant access, some Dockstore features, such as manual workflow
```

Comment:
Instead of "will not" maybe "may not" at least for workflow discovery. 
As per the user in the original discussion and as per our discussion, it seems that Dockstore may just work "enough" for regular github app use, publishing, etc.

Reply:
> Instead of "will not" maybe "may not" at least for workflow discovery. As per the user in the original discussion and as per our discussion, it seems that Dockstore may just work "enough" for regular github app use, publishing, etc.

I can soften it, but maybe also refer specifically to the "Discover Existing" functionality, rather than saying "workflow discovery", to avoid any confusion with being able to search or publish.

Reply:
Food for thought: yesterday, Charles and I were talking about the idea of authorizing everything as our GitHub App, which you could do with the `installationId`, I think, same way the webhook endpoints do.  If we recorded App installs, I think the `installationId` arrives with that.

Reply:
I think it would simplify things to remove OAuth.
I did an experiment in https://ucsc-cgl.atlassian.net/browse/SEAB-5915 and it seemed like most things just work using the GitHub app only